src/features/Account/ConfirmationEmailSentComponent.tsx

```diff
@@ -4,7 +4,7 @@ import * as React from 'react';
 import Helmet from 'react-helmet';
 
 import { APIResponse, Auth } from '../../api';
-import constructionSVG from '../../assets/images/construction.svg';
+import telescopeSVG from '../../assets/images/telescope.svg';
```

Comment:
I think we just used the svg name for Profile page like `Bulby` so would be nice to stay consistent here

Reply:
also for images, we've been order imports by putting assets at the bottom in their own group, see `ManageAccountContainer.tsx`